block/stream/output/state_changes.proto

```diff
@@ -553,24 +553,24 @@ message MapChangeValue {
          * An updated set of staking information for all nodes in
          * the address book.
          */
-        proto.StakingNodeInfo staking_node_info_value = 33;
+        proto.StakingNodeInfo staking_node_info_value = 10;
```

Comment:
Thanks for renumbering these; I completely missed this one.

Reply:
Good catch